rpcd/playbooks/roles/rpc_support/tasks/main.yml

```diff
@@ -41,11 +41,11 @@
 
 - include: support_key_pair.yml
   when: >
-    inventory_hostname == groups['utility'][0]
+    groups['nova_api_os_compute']|length > 0 and inventory_hostname == groups['utility'][0]
```

Comment:
I just finished building a standalone swift environment using `groups['nova_api_os_compute'][0] is defined`, and it seems to skip the relevant bits.

What I haven't done is build a full environment, and test to see if those same bits don't get skipped.